### PR TITLE
feat: Initial OJS Support

### PR DESCRIPTION
**Pull request by GordonSmith**
Use loose es6 files by default

Signed-off-by: Gordon Smith <GordonJSmith@gmail.com>

## Checklist:
- [x] The commit message is properly formatted and free of typos.
  - [x] The commit message title makes sense in a changelog, by itself.
  - [ ] The commit message includes a "fixes" reference if appropriate.
  - [x] The commit is signed.
- [x] The change has been fully tested:
  - [ ] I have viewed all related gallery items
  - [ ] I have viewed all related dermatology items
- [x] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised new issues to address them separately

### CODE REVIEW

**Comment by GordonSmith** [comment]
@jbrundage please review

File: demos/storyboard/src/app.ts [hunk 1]
```diff
@@ -100,11 +105,11 @@ export class App extends Border2 {
         if (!samples[defaultSelection]) {
             if (defaultSelection[0] === "@") {
                 // @lzxue/the-world-grid-map-use-l7
-                inspect(defaultSelection).then(md => {
-                    samples[defaultSelection] = md;
-                });
+                // inspect(defaultSelection).then(md => {
+                //     samples[defaultSelection] = md;
+                // });
```
Comment (jbrundage) [inline demos/storyboard/src/app.ts:110]: Was this intentional?

File: demos/storyboard/src/app.ts [hunk 2]
```diff
@@ -168,23 +187,33 @@ export class App extends Border2 {
...
     }
 
-    updateToolbar() {
-        this._buttonGenerate.enabled(this._mdEditor.text() !== this._omd.markdown()).lazyRender();
+    render(callback?: (w: Widget) => void): this {
+        super.render(w => {
+            if (this.renderCount() === 1) {
+                setTimeout(() => {
+                    this.generate();
+                }, 100);
```
Comment (jbrundage) [inline demos/storyboard/src/app.ts:210]: Why is setTimeout needed here?

File: packages/observable-md/src/util.ts [hunk 3]
```diff
@@ -54,3 +65,33 @@ export function encodeMD(str: string) {
         .split("$").join("\\$")
         ;
 }
+
+export function encodeBacktick(str: string) {
+    return str
+        .split("`").join("\\`")
```
Comment (jbrundage) [inline packages/observable-md/src/util.ts:71]: Why two backslashes here but one backslash on line 57?

File: demos/storyboard/src/app.ts (hunk 1 above)
Comment (GordonSmith) [reply demos/storyboard/src/app.ts:110]: Yes - I needed to do some quick fixes to the App to get the PR to compile!

File: demos/storyboard/src/app.ts (hunk 2 above)
Comment (GordonSmith) [reply demos/storyboard/src/app.ts:210]: Can't remember.  Will revisit

File: packages/observable-md/src/util.ts (hunk 3 above)
Comment (GordonSmith) [reply packages/observable-md/src/util.ts:71]: Its needed to make the following (broken code) work:
``````javascript
const innerStr = "`x`";
eval(`md\`${x}\`);
``````

Comment (GordonSmith) [reply packages/observable-md/src/util.ts:71]: Compared to:
```
const innerStr = "`x`";
eval(`${x}`);
```

**Pull request merged**